ocaml/xapi-guard/src/varstored_interface.ml

```diff
@@ -122,16 +126,8 @@ let logout session_id =
           Lwt.fail e
       )
 
-let cache = SessionCache.create ~login ~logout
-
 let shutdown = Lwt_switch.create ()
 
-let () =
```

Comment:
Why isn't this needed?

Reply:
It got moved into main.ml. There is no longer a global `cache` here, so there can't be a global shutdown hook either. The global cache is now declared in main.ml.
This is done so that the unit test can create and tear down the Lwt switch within each test (and test that this teardown actually works) instead of relying on a global (which could only test it once/process)

Reply:
Oh, you're right, I missed that piece of code in main.ml